data/json/recipes/ammo/rifle.json

```diff
@@ -177,6 +177,47 @@
       [ [ "copper", 3 ] ]
     ]
   },
+  {
+    "result": "reloaded_4570_sp",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_RIFLE",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "skills_required": [ "gun", 5 ],
+    "time": 2000,
+    "batch_time_factors": [ 60, 5 ],
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 4 ] ],
+    "charges": 1,
+    "using": [ [ "bullet_forming", 15 ], [ "ammo_bullet", 8 ] ],
+    "components": [
+      [ [ "4570_casing", 1 ] ],
+      [ [ "lgrifle_primer", 1 ] ],
+      [ [ "gunpowder", 15 ] ],
```

Comment:
This seems like a very old cartridge, so I'd consider adding blackpowder as an alternative here (same for recipe below).

Reply:
While it has its roots in black powder, the modern .45-70 loadings that produce the power I've attributed to it in-game are strictly smokeless powder.

I thought about doing a black powder load as a separate recipe, as from a realism standpoint, the resulting round would be MUCH less powerful.  But that would open a can of worms since you could theoretically do that with many non semi-auto rounds like .357 and .44 Magnum, so I figured that might be better under a separate PR.

Reply:
See below. Don't be afraid to add black powder variants, they're actually nigh identical to the commercially common loads. There's a PR open now that handles .357, 38 and 44 magnum loads, mostly using reloading data from Go-ex.